testing/lsmash/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Alexander Edland <alpine@ocv.me>
+# Maintainer: Alexander Edland <alpine@ocv.me>
+pkgname=lsmash
+pkgver=2.14.5
+pkgrel=0
+pkgdesc="MP4 and MOV muxer/demuxer"
+url="https://github.com/l-smash/l-smash"
+arch="all"
+license="ISC"
+options="!check"
+subpackages="$pkgname-dev"
+source="lsmash-$pkgver.tar.gz::https://github.com/l-smash/l-smash/archive/v$pkgver.tar.gz"
+builddir="$srcdir/l-smash-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--prefix=/usr \
+		--extra-cflags=-fPIC \
+		--disable-static \
```

Comment:
Is there a reason for disabling static builds here?

Reply:
I thought that was an Alpine policy, forcing everything to dynamic link -- the package and everything that depends on it of course builds just fine with both dynamic and static libs available for linking, so I'll remove that flag.